### ISSUE TITLE
How to use it?

### ISSUE THREAD

**Issue by suntong** [LeKovr/teleproxy#6]
How to use it?

Is there any usage summary like this, https://github.com/go-easygen/wireframe#-wireframe? 

How do you config & start it? Thx.

**Comment by suntong** [comment]
Thanks!

I think one thing is missing, at github.com/LeKovr/teleproxy, 

    ln -s cmd/teleproxy/teleproxy

In order to use `make run`

**Comment by LeKovr** [comment]
What I'm doing wrong?
```
$ cd $GOPATH/src/github.com/LeKovr/teleproxy
$ make run
[ -d .git ] && GH=`git rev-parse HEAD` || GH=nogit ; \
  GOOS= GOARCH= go build -v -o teleproxy -ldflags \
  "-X main.Build= -X main.Commit=$GH" ./cmd/teleproxy
./teleproxy --log_level debug --group $GROUP --token $TOKEN --command ./commands.sh --template messages.tmpl
[ debug ] 2018/05/13 00:23:18 main.go:63: teleproxy v 0.4. Telegram proxy bot
...

**Comment by suntong** [comment]
My bad. I built myself and didn't look closely how the build was done by `make`. Thanks for  the explanation.

**Comment by suntong** [comment]
Umm..., is your above the output before fix cddd9f6 or after? 

After the fix to this issue, #6, this is the command I'm seeing:

```sh
$ make run -n 
./teleproxy --log_level debug --group $GROUP --token $TOKEN --template $TEMPLATE --command ./commands.sh --template messages.tmpl
```

that doesn't look right, does it? (two `--template` last being the Russion version)

**Comment by suntong** [comment]
Oh, one more thing that's a typo

[ error ] 2018/05/12 22:13:19 main.go:131: Template load error: open messages.en.html: no such file or directory

I.e., the `make` default should be `messages.en.tmpl` instead of `messages.en.html`

**Comment by suntong** [comment]
Back to two `--template`, I thought it isn't right so I started from bash myself using:

    ./teleproxy --log_level debug --group $GROUP --token $TOKEN --template $TEMPLATE --command ./commands.sh

with `TEMPLATE=messages.en.tmpl`

however, during execution, I'm seeing:

```
[  warn ] 2018/05/12 22:23:13 app.go:46: template info exec error: template: no template "messages.tmpl" associated with template "messages.tmpl"
[  warn ] 2018/05/12 22:23:13 app.go:46: template message exec error: template: no template "messages.tmpl" associated with template "messages.tmpl"
```

that doesn't look right, does it?

I didn't see any messages being forwarded to my designated group, FYI. 

What do you get when you try with the English template?

**Comment by LeKovr** [comment]
Offer about `ln -s cmd/teleproxy/teleproxy` still needs explain. Others was fixed in e965c6f and 420f3de

**Comment by suntong** [comment]
> Offer about ln -s cmd/teleproxy/teleproxy still needs explain

I built myself teleproxy, with go, thus `teleproxy` is under `cmd/teleproxy` as `cmd/teleproxy/teleproxy`, not under `github.com/LeKovr/teleproxy`. Thus I thought `ln` is necessary, and I didn't look closely how the build was done by `make` to build it under `github.com/LeKovr/teleproxy`. 

Hope clearer this time. 
Thanks to other fixes!
